subiquity/server/controllers/filesystem.py

```diff
@@ -371,17 +405,64 @@ async def guided_GET(self, wait: bool = False) -> GuidedStorageResponse:
         se = None
         if self._system is not None:
             se = self._system.storage_encryption
+            for _, offset, size in self._offsets_and_sizes_for_system():
```

Comment:
I'm nitpicking at this point but we don't need to iterate to assign the min size if we only care about  the last entry.  This structure does give the reader the impression that we need to accumulate all the values.

Reply:
fair enough.

Reply:
LGTM